client/reader/controller.js

```diff
@@ -58,14 +58,11 @@ function trackScrollPage( path, title, category, readerView, pageNum ) {
 
 // Listen for route changes and remove the full post dialog when we navigate away from it
 pageNotifier( function removeFullPostOnLeave( newContext, oldContext ) {
-	if ( ! oldContext ) {
```

Comment:
I removed this check because I think it is redundant and was accidentally left. On line 63, we firstly check if `oldContext` exists before using it.